src/protocols/secure_channel/CASEServer.cpp

```diff
@@ -60,15 +54,6 @@ CHIP_ERROR CASEServer::InitCASEHandshake(Messaging::ExchangeContext * ec)
 {
     ReturnErrorCodeIf(ec == nullptr, CHIP_ERROR_INVALID_ARGUMENT);
 
-#if CONFIG_NETWORK_LAYER_BLE
-    // Close all BLE connections now since a CASE handshake has been initiated.
-    if (mBleLayer != nullptr)
-    {
-        ChipLogProgress(Discovery, "CASE handshake initiated, closing all BLE Connections");
```

Comment:
Still needs the followup for the "commissioning window timer expires before a PASE session is established" case, right?

Reply:
Tracking issue here: https://github.com/project-chip/connectedhomeip/issues/17508